lib/middleware/domain.js

```diff
@@ -26,7 +26,10 @@
 var domain = require('domain'),
```

Comment:
CNR entry corresponding to the "recovery" of the functionality should be included.